doc/nrf/security/ap_protect.rst

```diff
@@ -197,16 +216,98 @@ This is the default setting in the |NCS|.
 
 You can start debugging the firmware without additional steps needed.
 
-Once you are done debugging, run the following command to enable the access port protection:
+.. _app_approtect_uicr_approtect:
+
+Enabling AP-Protect by locking the UICR.APPROTECT register
+==========================================================
+
+For the devices that are in a production environment, you are highly recommended to lock the UICR.APPROTECT register to prevent unauthorized access to the device.
```

Comment:
```suggestion
For devices that are in a production environment, it is highly recommended to lock the ``UICR.APPROTECT`` register to prevent unauthorized access to the device.
```